screenshots/test.py

```diff
@@ -0,0 +1,57 @@
+from seltest import url, waitfor, hide, Base
+from selenium.webdriver.support.ui import Select, WebDriverWait
+import sys
+
+PILEUP_RENDERED_JS = '''
+return document.querySelector('.track.pileup canvas') && document.querySelector('.track.pileup canvas').height > 100
+'''
+
+def _pileup_is_rendered(driver):
+    return driver.execute_script(PILEUP_RENDERED_JS)
+
+
+def wait_for_pileup(driver):
+    driver.implicitly_wait(0)
```

Comment:
I cribbed that from [here][1]. I'm not sure why, but it was necessary—the test didn't wait properly if I didn't include it.

[1]: https://github.com/hammerlab/seltest/blob/c73caeee879a790a6a21396e502710802f0430f3/seltest/seltest.py#L188-L192